incubator/hnc/test/e2e/quickstart_test.go

```diff
@@ -249,4 +249,39 @@ spec:
 		RunShouldContain(expected, defTimeout, "kubectl hns tree", nsTeamA)
 		RunShouldContain("ActivitiesHalted (ParentMissing):", defTimeout, "kubectl hns describe", nsStaging)
 	})
+
+	It("Should demonstrate exceptions", func() {
+		// set up initial structure
+		CreateNamespace(nsOrg)
+		CreateSubnamespace(nsTeamA, nsOrg)
+		CreateSubnamespace(nsTeamB, nsOrg)
+
+		MustRun("kubectl -n", nsOrg, "create secret generic my-secret --from-literal=password=iamacme")
+		// allow secret to propagate
+		MustRun("kubectl hns config set-resource secrets --mode Propagate --force")
+		// wait 2 seconds to give time for secret to propogate
+		time.Sleep(2 * time.Second)
```

Comment:
Is this necessary? `RunShouldContain` should wait, right? Was it just not waiting long enough?

Reply:
+1

Reply:
Removed. It was a check for `RunShouldNotContain()` so I added the wait to confirm the object is nor propagated to the subnamespaces. Now it's not needed anymore